src/explorer.api/Explorers/ExplorerImpl.cs

```diff
@@ -0,0 +1,41 @@
+namespace Explorer
+{
+    using System.Collections.Concurrent;
+    using System.Threading.Tasks;
+
+    using Aircloak.JsonApi;
+    using Aircloak.JsonApi.ResponseTypes;
+
+    internal abstract class ExplorerImpl
```

Comment:
`ExplorerImpl` is pretty unfortunate name, since it is an abstract class, not an implementation. I would call it simply `Explorer` or `ExplorerBase`.

Reply:
Yeah the naming is a bit crappy. How about changing `ExplorerImpl` to `Exploration` and `ColumnExplorer` can be `Explorer`? That way, the Explorer manages a number of explorations... makes sense?

Reply:
Yes, sounds good.

Reply:
Yes, this sounds good.

Reply:
Done: 6dc629ebd0b123fadbee1c94a7b6b09f9d7bf6b5